apps/docs/src/app/core/component-docs/dialog/examples/dialog-configuration/dialog-configuration-example.component.ts

```diff
@@ -3,7 +3,19 @@ import { DialogService } from '@fundamental-ngx/core/dialog';
 
 @Component({
     selector: 'fd-dialog-configuration-example',
-    templateUrl: './dialog-configuration-example.component.html'
+    templateUrl: './dialog-configuration-example.component.html',
+    styles: [
+        `
+            .fd-button {
```

Comment:
Since the styles are repeated, you can create common scss file and use it for all examples.